tests/BenchmarkDotNet.Tests/BenchmarkDotNet.Tests.csproj

```diff
@@ -40,4 +40,12 @@
   <ItemGroup>
     <EmbeddedResource Include="Portability\Cpu\TestFiles\*.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Portability\Memory\TestFiles\SysctlMemory.txt" />
+    <None Remove="Portability\Memory\TestFiles\VmStatMemory.txt" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Portability\Memory\TestFiles\SysctlMemory.txt" />
+    <EmbeddedResource Include="Portability\Memory\TestFiles\VmStatMemory.txt" />
+  </ItemGroup>
```

Comment:
Please, embed the whole folder (`<EmbeddedResource Include="Portability\Memory\TestFiles\*.txt" />`)

Reply:
Done